Content.Shared/Preferences/Loadouts/Effects/JobRequirementLoadoutEffect.cs

```diff
@@ -1,6 +1,8 @@
 using System.Diagnostics.CodeAnalysis;
+using Content.Shared.CCVar;
```

Comment:
You'll want to also tag changes if we import anything
```suggestion
using Content.Shared.CCVar; // Moffstation
```